docs/content/zh/monitor/apply_monitoring.md

```diff
@@ -0,0 +1,139 @@
+# 开启监控
+
+## 简介
+
+[Prometheus](https://prometheus.io/) 基于文本的暴露格式，已经成为云原生监控领域事实上的标准格式。
+
+RadonDB MySQL 监控引擎基于 [Prometheus MySQLd Exporter](https://github.com/prometheus/mysqld_exporter) 定义。通过 `mysqld-exporter` 抓取 RadonDB MySQL 服务指标，再通过接入第三方应用平台实现监控指标可视化。
+
+本教程演示如何开启 RadonDB MySQL 监控指标。
+
+## 约束限制
+
+- RadonDB MySQL Operator 2.1.0及以上版本，支持自动开启监控引擎。
+
+## 准备工作
+
+- 已准备可用 Kubernetes 或 KubeSphere 集群。
+
+## 部署步骤
+
+### 步骤 1: 配置 serviceMonitor
+
+`serviceMonitor` 是定义 RadonDB MySQL Operator 自动监控引擎的参数，开启后将自动绑定 `mysqld_exporter` 与 Prometheus。
+
+`serviceMonitor`定义包含如下字段：
+
+```bash
+serviceMonitor:
+  enabled: true
+  ## Additional labels for the serviceMonitor. Useful if you have multiple prometheus operators running to select only specific ServiceMonitors
+  # additionalLabels:
+  #   prometheus: prom-internal
+  interval: 10s
+  scrapeTimeout: 3s
+  # jobLabel:
+  # targetLabels:
+  # podTargetLabels:
+  namespaceSelector:
+    any: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/managed-by: mysql.radondb.com
+      app.kubernetes.io/name: mysql
+```
+
+您可以在 `charts/mysql-operator/values.yaml` 文件中配置`serviceMonitor`。
+
+- 新部署 Operator 时， `serviceMonitor.enabled` 默认为**true**，表示默认开启。
+- 已部署 Operator 的集群，需重新部署 Operator。
+
+### 步骤 2: 配置 metricsOpts
+
+`metricsOpts` 是 `mysql_v1alpha1_cluster.yaml` 文件中定义 RadonDB MySQL 集群监控的参数。`metricsOpts.enabled`默认为 **false**，需手动设置为**true**。
+
+`metricsOpts` 定义包含如下字段：
+
+```bash
+metricsOpts:
+    enabled: false  
+    image: prom/mysqld-exporter:v0.12.1
+
+    resources:
+      requests:
+        cpu: 10m
+        memory: 32Mi
+      limits:
+        cpu: 100m
+        memory: 128Mi
+```
+
+可选择设置`metricsOpts.enabled` 状态为**true**，开启集群监控功能；并可通过设置资源参数值，定义监控容器资源配额大小。
+
+文件参数修改完成后，使用如下指令应用配置，部署/更新集群回显信息如下：
+
+```bash
+$ kubectl apply -f config/sample/mysql_v1alpha1_cluster.yaml
+cluster.mysql.radondb.com/sample created/configured
+```
+
+## 查看监控服务
+
+### 通过客户端查看
+
+开启监控后，将自动创建一个监控服务和 serviceMonitor。 您可以通过如下指令查看集群监控信息。
```

Comment:
Not the serviceMonitor created after the monitoring is turned on, but when the operator is deployed

Reply:
done